.cbmc-batch/jobs/Makefile.common

```diff
@@ -31,15 +33,22 @@ CFLAGS += $(CFLAGS2) $(DEF) $(OPT)
 
 LINKFARM = $(abspath ./link-farm)
 
+HARNESS_NAME ?= $(ENTRY)_harness.c
+
 ################################################################
-CBMCFLAGS += \
-	--bounds-check \
-	--pointer-check \
-	--signed-overflow-check \
-	--unsigned-overflow-check \
-	--pointer-overflow-check \
-	--div-by-zero-check \
-	--unwinding-assertions
+CBMCFLAGS += --bounds-check
+CBMCFLAGS += --div-by-zero-check
+CBMCFLAGS += --float-overflow-check
+CBMCFLAGS += --memory-leak-check
```

Comment:
Since many of the operations in cipher deal with allocation and deallocation, I thought it would be good to also check for memory leaks. But this means that we'll need to do explicit clean-ups in the harnesses.